alembic/versions/2016031317_add_bundle_contents_index_40d61632fd13.py

```diff
@@ -0,0 +1,22 @@
+"""Add bundle contents index
```

Comment:
I don't see the word index as causing confusion. It's going to take a bunch of time to switch all my code to use the new name. Unless, you're really against using the word index, I'm going to leave it as is.

The name index, in general, is appropriate for what this is. It's a table listing the contents of what's on the file system.